wrangler-service/src/main/java/io/cdap/wrangler/service/directive/WorkspaceHandler.java

```diff
@@ -117,9 +117,6 @@ public class WorkspaceHandler extends AbstractDirectiveHandler {
   @SuppressWarnings("unused")
   private Metrics metrics;
 
-  private static final String APPLY_RECIPE_SUCCESS_METRIC = "recipe.apply.success";
-  private static final String APPLY_RECIPE_FAILURE_METRIC = "recipe.apply.failure";
```

Comment:
Which file/function are these metrics being captured right now.

Reply:
CDAP by default captures metrics for all HTTP API calls. `io.cdap.cdap.common.metrics.MetricsReporterHook` in `cdap` repo is responsible for capturing these metrics.

Reply:
But would it be able to capture specific metric like count of directives in a recipe. I understand it could capture "ApplyRecipe" as that is a http call specific metric.